charts/astronomer/templates/registry/registry-configmap.yaml

```diff
@@ -34,6 +34,10 @@ data:
     {{- else }}
       filesystem:
         rootdirectory: /var/lib/registry
+    {{- end }}
+    {{- if .Values.registry.redirectDisabled }}
```

Comment:
Should this be `.Values.registry.redirect.enabled` ? That seems more consistent with the data structure and naming that we use elsewhere.

Reply:
That was my initial instinct, but will that confuse the user as values will have `redirect.enabled` but the registry config map will have it as `redirect.disable`?

Reply:
@neel-astro i'm missing some context. where is this `redirect.disable` in the registry config map? If possible we should also change that to `enabled`. `disabled = False` is a double negative, which is harder to grok.

https://schneide.blog/2014/08/03/dont-ever-not-avoid-negative-logic/

Reply:
By `redirect.disable` I meant `disable` under `redirect`section in yaml ([code reference](https://github.com/astronomer/astronomer/pull/1430/files#diff-087010573e7ff5d6f9d72b6fc63bf83256d1e52d9e86611fa177bdee5da84cafR40)).
We won't be able to change that to `enable` since that is defined on the docker registry side (Reference: https://docs.docker.com/registry/configuration/#redirect)

Reply:
SMH. ok, well it would be better for us to keep what they are using. i think we should use `redirect.disable` though instead of `redirectDisabled`. what do you think?

Reply:
sure `redirect.disable` feels better than `redirectDisabled`, I will make the required changes.